src/cdk-experimental/testing/protractor/protractor-element.ts

```diff
@@ -55,4 +55,8 @@ export class ProtractorElement implements TestElement {
     const classes = (await this.getAttribute('class')) || '';
     return new Set(classes.split(/\s+/).filter(c => c)).has(name);
   }
+
+  async getPropertyValue(name: string): Promise<any> {
```

Comment:
Could these APIs use `unknown` instead of `any`?

Reply:
Good idea. I'd also vote for that.